Tests/SwiftDocCTests/Rendering/DefaultAvailabilityTests.swift

```diff
@@ -33,11 +33,24 @@ class DefaultAvailabilityTests: XCTestCase {
     // Test whether the default availability is loaded from Info.plist and applied during render time
     func testBundleWithDefaultAvailability() throws {
         // Copy an Info.plist with default availability
-        let (url, bundle, context) = try testBundleAndContext(copying: "TestBundle", excludingPaths: [], codeListings: [:]) { (url) in
+        let (_, bundle, context) = try testBundleAndContext(copying: "TestBundle", excludingPaths: [], codeListings: [:]) { (url) in
             try? FileManager.default.removeItem(at: url.appendingPathComponent("Info.plist"))
             try? FileManager.default.copyItem(at: self.infoPlistAvailabilityURL, to: url.appendingPathComponent("Info.plist"))
+            
+            let myKitDocExtensionFile = url.appendingPathComponent("documentation", isDirectory: true).appendingPathComponent("mykit.md")
+            var myKitDocExtension = try String(contentsOf: myKitDocExtensionFile)
+            
+            let firstNewLineIndex = try XCTUnwrap(myKitDocExtension.firstIndex(of: "\n"))
+            myKitDocExtension.insert(contentsOf: """
```

Comment:
Nice test!

Maybe we should add a comment here explaining why we're explicitly customizing the display name in the default availability test though? I think I'd be a little confused looking back on it.